src/applications/vaos/utils/data.js

```diff
@@ -22,6 +28,14 @@ function getRequestedDates(data) {
   );
 }
 
+function getUserMessage(data) {
+  const label = PURPOSE_TEXT.find(
+    purpose => purpose.id === data.reasonForAppointment,
+  ).short;
+
+  return `${label} - ${data.reasonAdditionalInfo}`;
```

Comment:
Is the ` - ` between label and additional info formatting copied from legacy vaos?  We might need to do some testing around the max characters and formatting allowed here.  In [this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/2301) that I worked on for the character limit, one of the AC says:

> Chosen 'Purpose of visit' selection is pre-pended to the 'reason for appointment' text in the form data that's submitted like so: `Purpose of visit: ${selection}; Veteran free text goes here.`

On another note, if `Purpose of visit: ` is required to be prepended, we may have to update my character limit code to take it into account.

Reply:
I think I need to switch to `: ` instead of ` - `, I should have gone back and looked to make sure I was choosing the right separator.

Reply:
Adding "Purpose of visit" seems like cutting into the available characters for veterans even further, maybe we can open a ticket for that and decide later to change it.

Reply:
Yeah, prepending that seems odd, but we've seen some weirder stuff.